src/main/resources/assets/js/launcher/main.ts

```diff
@@ -309,6 +310,7 @@ class Launcher {
     private closeLauncherPanel = (skipTransition?: boolean): void => {
         document.removeEventListener('click', this.onLauncherClick);
         this.launcherMainContainer.setAttribute('hidden', 'true');
+        this.launcherMainContainer.style.visibility = 'hidden';
```

Comment:
We have an animation of opening/closing launcher menu, and immediately hiding/showing it will break that behavior